README.md

```diff
@@ -257,7 +257,7 @@ Used to determine how many digits after the decimal will be allowed. For instanc
 Type: `Boolean`
 Default: `false`
 
-`true` enables additional debugging information in the output file as CSS comments
+`true` enables additional debugging information (line number and source scss file reference, similar to :line_comments in compass) in the output file as CSS comments
```

Comment:
`:line_numbers` is the Ruby Sass one (with `:line_comments`  as an alias) http://sass-lang.com/documentation/file.SASS_REFERENCE.html#options
Compass is depreciated now, so probably shouldn't be mentioned